packages/wasm/src/tx-planner.ts

```diff
@@ -58,8 +61,12 @@ export class TxPlanner {
     this.wasmPlanner.ics20_withdrawal(withdrawal.toJson());
   }
 
-  delegate(delegation: TransactionPlannerRequest_Delegate): void {
-    this.wasmPlanner.delegate(delegation.amount?.toJson(), delegation.rateData?.toJson());
+  delegate(delegate: TransactionPlannerRequest_Delegate): void {
+    this.wasmPlanner.delegate(delegate.amount?.toJson(), delegate.rateData?.toJson());
+  }
```

Comment:
Only change here was renaming the `delegation` arg to `delegate` for consistency with the type name